loader/loader_test.go

```diff
@@ -1811,3 +1811,24 @@ services:
 `)
 	assert.NilError(t, err)
 }
+
+func TestLoadService(t *testing.T) {
+	file, err := os.CreateTemp(os.TempDir(), "test-compose-go")
```

Comment:
```suggestion
	file, err := os.CreateTemp("", "test-compose-go")
```
Because as the [docs](https://pkg.go.dev/os#CreateTemp) say `If dir is the empty string, CreateTemp uses the default directory for temporary files, as returned by TempDir`.

Reply:
good point!